### ISSUE TITLE
Valient useless after extreme early game

### ISSUE THREAD

**Issue by coyotesfrontier** [PorktoberRevolution/ReStocked#647]
As an unwritten rule, every part in Kerbal has its use, no matter how far you've progressed.

The Valient's ISP is way too low to be useful anywhere outside of extreme early game, but changing it to serve the role of a mid-range engine in addition to a starting one would fill a empty spot in the 1.25 lineup.

An ISP increase to somewhere between the Swivel and Terrier (probably 330) would give 1.25m a higher-thrust vacuum engine, like the Skipper or Rhino (Swivel sorta fills this role, but not really).

tl;dr, Valient is useless, but could fill a unique spot with a small change.

**Comment by TykoTek** [comment]
Valiant is definitely overshadowed by other engines later and that could be addressed by changing ISP. I think that would go contrary to the original intent for Valiant as laid down by Porkjet in [PartOverhauls](http://forum.kerbalspaceprogram.com/index.php?/topic/147582-update-12-pre-release-is-here/).

Since the the RS+ versions are based on the same models and stats as PartOverhauls I think it would be incongruous to start making changes to those established stats and would only cause confusion.

It's pretty common for older engines to receive block updates over time. Perhaps the solution would be to add a Valiant Block 2 engine further down the tech tree with improved stats? This is probably outside the scope for RS+ (i'll leave that decision to @ChrisAdderley), but it would be easy enough to use the +PART MM command to add one on your own.

@coyotesfrontier, I'm happy to help you on making up a new config if you decide to go that route.

**Comment by coyotesfrontier** [comment]
I've already made a patch that increases the issues, it's an incredibly simple change:
https://www.dropbox.com/sh/uyx4zimkrxwvz5l/AACAYTgwV_Oh9rghZer5JC_ka?dl=0

**Comment by TykoTek** [comment]
@coyotesfrontier glad that worked for you. Thanks for the feedback. Sorry we couldn't accommodate this request, but keep the suggestions coming.
